test/tools/libtesteth/TestHelper.cpp

```diff
@@ -637,6 +637,36 @@ string prepareLLLCVersionString()
     return "Error getting LLLC Version";
 }
 
+// A simple C++ implementation of the Levenshtein distance algorithm to measure the amount of
+// difference between two strings. https://gist.github.com/TheRayTracer/2644387
+size_t levenshtein_distance(const char* s, size_t n, const char* t, size_t m)
```

Comment:
I'd prefer this to be reformatted using our name conventions, so
```suggestion
size_t levenshteinDistance(char const* _s, size_t _n, char const* _t, size_t _m)
```